sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,93 @@
+---
+parent: 'common/test_cases/origin_built_release.yml'
+overrides:
+  sync:
+    - "openshift,ansible-service-broker=master"
```

Comment:
This should be in the extensions section so the repos synced is appended, not replaced.